homeassistant/components/gree/climate.py

```diff
@@ -160,7 +160,7 @@ def precision(self) -> float:
     @property
     def current_temperature(self) -> float:
         """Return the target temperature, gree devices don't provide internal temp."""
-        return self.target_temperature
+        return self.coordinator.device.current_temperature
```

Comment:
Is this a bug fix?

Reply:
Not particularly. Instead of using the property below which then reads from the device state, it reads directly from device state. This divergence from my testing/dev branch and here happened in previous merge requests, I can drop it if you don't want it, but it's been what I've been using in my development for the better part of the past year. Same behaviour either way.